docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '2.0'
+services:
+  server:
+    image: docker.pkg.github.com/capstoneris/css-backend/css-backend:master
```

Comment:
Better specify a container name

Reply:
Also, how is the server configured? How does it know how to reach the database server? There should probably be some environment variables for this?